File: Java/백준/Bronze/30018. 타슈/타슈.java
import java.util.Scanner;
import java.lang.Math;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int[] aa = new int[a];
        for (int i=0;i<a;i++){
            aa[i]=in.nextInt();
        }
        int[] bb = new int[a];
        for (int i=0;i<a;i++){
            bb[i]=in.nextInt();
        }
        int total = 0;
        for (int i=0;i<a;i++){
            total+= Math.abs(aa[i]-bb[i]);
        }
        System.out.println(total/2);
    }
}

File: C#/백준/Bronze/30018. 타슈/타슈.cs
using System;

namespace q{
    class w{
        static void Main(String[] args){
            int a = int.Parse(Console.ReadLine());
            int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int[] bb = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
            int total = 0;
            for (int i=0;i<a;i++){
                total+=Math.Abs(aa[i]-bb[i]);
            }
            Console.WriteLine(total/2);
        }
    }
}